src/main/java/slimeknights/tconstruct/items/FoodItems.java

```diff
@@ -1,96 +1,70 @@
-    registry.register(createEdible(TinkerFood.MUTTON_JERKY), "mutton_jerky");
-    registry.register(createEdible(TinkerFood.RABBIT_JERKY), "rabbit_jerky");
-    registry.register(createEdible(TinkerFood.FISH_JERKY), "fish_jerky");
-    registry.register(createEdible(TinkerFood.SALMON_JERKY), "salmon_jerky");
-    registry.register(createEdible(TinkerFood.CLOWNFISH_JERKY), "clownfish_jerky");
-    registry.register(createEdible(TinkerFood.PUFFERFISH_JERKY), "pufferfish_jerky");
-
-    registry.register(createEdible(TinkerFood.GREEN_SLIME_DROP), "green_slime_drop");
-    registry.register(createEdible(TinkerFood.BLUE_SLIME_DROP), "blue_slime_drop");
-    registry.register(createEdible(TinkerFood.PURPLE_SLIME_DROP), "purple_slime_drop");
-    registry.register(createEdible(TinkerFood.BLOOD_SLIME_DROP), "blood_slime_drop");
-    registry.register(createEdible(TinkerFood.MAGMA_SLIME_DROP), "magma_slime_drop");
+  /* Jerkies */
+  public static final ItemObject<EdibleItem> monster_jerky = ITEMS.register("monster_jerky", () -> new EdibleItem(TinkerFood.MONSTER_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> beef_jerky = ITEMS.register("beef_jerky", () -> new EdibleItem(TinkerFood.BEEF_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> chicken_jerky = ITEMS.register("chicken_jerky", () -> new EdibleItem(TinkerFood.CHICKEN_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> pork_jerky = ITEMS.register("pork_jerky", () -> new EdibleItem(TinkerFood.PORK_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> mutton_jerky = ITEMS.register("mutton_jerky", () -> new EdibleItem(TinkerFood.MUTTON_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> rabbit_jerky = ITEMS.register("rabbit_jerky", () -> new EdibleItem(TinkerFood.RABBIT_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> fish_jerky = ITEMS.register("fish_jerky", () -> new EdibleItem(TinkerFood.FISH_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> salmon_jerky = ITEMS.register("salmon_jerky", () -> new EdibleItem(TinkerFood.SALMON_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> clownfish_jerky = ITEMS.register("clownfish_jerky", () -> new EdibleItem(TinkerFood.CLOWNFISH_JERKY, TinkerRegistry.tabGeneral));
+  public static final ItemObject<EdibleItem> pufferfish_jerky = ITEMS.register("pufferfish_jerky", () -> new EdibleItem(TinkerFood.PUFFERFISH_JERKY, TinkerRegistry.tabGeneral));
+
+  /* Slime Balls are edible, believe it or not */
+  public static final EnumObject<SlimeBlock.SlimeType, Item> slime_ball;
+  static {
+    EnumObject<SlimeBlock.SlimeType,EdibleItem> tinkerSlimeballs = ITEMS.registerEnum(SlimeBlock.SlimeType.TINKER, "slime_ball", (type) -> new EdibleItem(type.getSlimeFood(type), TinkerRegistry.tabGeneral));
```

Comment:
Since we do this twice, I almost wonder if we should have a more streamlined way to do it.

* One option is passing a `Consumer<Map<T,Supplier<? extends I>` as an optional parameter to `registerEnum`, which will be called to add extra data to the map before returning.
* Another option which may be a bit easier to write is adding a method that returns the map directly, either from `EnumObject` (as a copy) or the deferred register, so we can add extra items.

Reply:
Yeah, that could be helpful, but I think its outside my grok atm.

Reply:
Yeah, just leave it and I can deal with it later if it becomes common then.